custom-login/src/components/Profile.vue

```diff
@@ -57,7 +57,8 @@ export default {
     }
   },
   async created () {
-    this.claims = await Object.entries(await this.$auth.getUser()).map(entry => ({ claim: entry[0], value: entry[1] }))
```

Comment:
the map function here might be the cause

Reply:
That data returned from the user info endpoint isn't a 1-1 mapping of the token info. for example something like this:
```javascript
const user = await this.$auth.getUser()
console.log("user: ", JSON.stringify(user))
```
Does not return all of the claims in the Id Token, it's missing all of the JWT related claims, the `amr` claim, `auth_time`, and any custom claims added via an Okta Token Inline Hook